qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/RecoveryIT.java

```diff
@@ -472,6 +472,62 @@ public void testUpdateDoc() throws Exception {
         }
     }
 
+    /** Ensure that we can always execute delete-by-query regardless of the version of cluster */
```

Comment:
nit: It seems a little odd to have these tests in `RecoveryIT`. Given that this is 6.8 only, I think it is OK, but if it is easy to add into a separate test class I would prefer that (could be named `BulkByScrollIT`).

Reply:
I've put them there as testUpdateDoc (which has same purpose) is also already there. Given that this is the 6.8 branch, I will leave as is.